doc/source/whatsnew/v0.24.0.rst

```diff
@@ -1295,6 +1295,7 @@ Performance Improvements
 - Improved performance of iterating over a :class:`Series`. Using :meth:`DataFrame.itertuples` now creates iterators
   without internally allocating lists of all elements (:issue:`20783`)
 - Improved performance of :class:`Period` constructor, additionally benefitting ``PeriodArray`` and ``PeriodIndex`` creation (:issue:`24084` and :issue:`24118`)
+- Improved performance of tz-aware :class:`DatetimeArray` binary operations
```

Comment:
Reference your PR as the issue number in cases like these (if there are any outstanding issues related to this, that would be great, as we can also close those out).

Reply:
Don't know the number until I submit! Will update in next push - not aware of any open issues.

Reply:
Sounds good.  Just wanted to make sure we had that filled out 👍

Reply:
Done